actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -73,6 +70,9 @@ def flush_newline_if_pending(src)
         end
       end
 
+      # For backwards compatibility
+      Erubis = Erubi
```

Comment:
Here are the public instance methods that Erubis::Eruby has that Erubi::Engine lacks (omitting the ones added by Rails ERB Handler):

* add_expr
* add_expr_debug
* add_expr_escaped
* add_expr_literal
* add_preamble
* add_stmt
* add_text
* convert
* convert!
* convert_input
* def_method
* escape
* escape=
* escaped_expr
* escapefunc
* escapefunc=
* filename=
* init_converter
* init_evaluator
* init_generator
* pattern
* pattern=
* postamble
* postamble=
* preamble
* preamble=
* process
* process_proc
* result
* src=
* trim
* trim=

Most of these have no equivalent in Erubi, and are not used.  Certainly the setter methods couldn't have an equivalent, as Erubi::Engine.new returns a frozen object.

If you really wanted to deprecate, you'd probably want to ship with both Erubis and Erubi, and deprecate Erubis.new.  I can prepare a commit that does that if you want.  AFAICT, nothing accesses Erubis directly outside of the Rails ERB Handler except for a test helper (`actionview/test/template/erb/helper.rb`)